mmocr/core/evaluation/utils.py

```diff
@@ -136,24 +136,32 @@ def points2polygon(points):
     return plg(point_mat)
 
 
-def poly_intersection(poly_det, poly_gt, invalid_ret=0, return_poly=False):
+def poly_intersection(poly_det, poly_gt, invalid='allow', return_poly=False):
```

Comment:
Invalid parameter can be bool type. Using string type need to set scope and assert invalid parameter is set proper